readme.rst

```diff
@@ -620,14 +620,75 @@ Function Reference:
 
 `fix_variables(bqm, sampling_mode=True) <https://docs.ocean.dwavesys.com/projects/dimod/en/latest/reference/bqm/generated/dimod.roof_duality.fix_variables.html?highlight=fix_variables#dimod.roof_duality.fix_variables>`_
 
+Other Tools
+===============================
+
+qbsolv
+-----------------
+
+The ``qbsolv`` utility has for all intents and purposes been replaced with the ``dwave-hybrid`` framework in Ocean
+(it is possible to build a ``qbsolv`` replica with Ocean). Read more about `D-Wave Hybrid <https://docs.ocean.dwavesys.com/projects/hybrid/en/stable/>`_.
+However, `qbsolv <https://docs.ocean.dwavesys.com/projects/qbsolv/en/latest/index.html#qbsolv>`_ has also been made available in Ocean.
+
+Ocean Tools
+.........................
+
+.. code:: python
+
+  from dwave_qbsolv import QBSolv
+  Q = {(0, 0): 1, (1, 1): 1, (0, 1): 1}
+  response = QBSolv().sample_qubo(Q)
+  print("samples=" + str(list(response.samples())))
```

Comment:
why not just `print(response)`?

Reply:
This is taken directly from the example in the link.

Reply:
better yet. why have print statements here at all!!? none of the other examples do.